vm.go

```diff
@@ -351,6 +353,54 @@ func (vm *VirtualMachineSpec) ExecuteCommand(command string) error {
 	return nil
 }
 
+// RunCommand executes a command on the Virtual Machine
+func (vm *VirtualMachineSpec) RunCommand(command []string) (output string, err error) {
+	ctx, cancel := context.WithTimeout(context.Background(), 60*time.Minute)
+	defer cancel()
+	system, err := hcs.OpenComputeSystem(ctx, vm.ID)
+	if err != nil {
+		return "", err
+	}
+	defer system.Close()
+
+	if system.OS() != "linux" {
+		return "", ErrNotSupported
+	}
+
+	params := &hcsschema.ProcessParameters{
+		CommandArgs:      command,
+		WorkingDirectory: "/",
+		Environment:      map[string]string{"PATH": "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"},
+		CreateStdInPipe:  false,
+		CreateStdOutPipe: true,
+		CreateStdErrPipe: false,
```

Comment:
>false [](http://example.com/codeflow?start=20&length=5)

I think stderr is required (or caller decide).

Reply:
keep Stderr. in case of error, we can report stderr message...